pkg/db/models/releases.go

```diff
@@ -0,0 +1,35 @@
+package models
+
+type ReleaseTag struct {
```

Comment:
Can you clarify why we need two very similar models?

Reply:
We don't, I'm not sure why I duplicated it TBH.  I'll remove the bigquery one.